pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -123,7 +124,7 @@ def work(self, response_dict=None):
         self._do_catch(pokemon, encounter_id, catch_rate_by_ball, is_vip=is_vip)
 
         # simulate app
-        time.sleep(5)
+        action_delay(self.config.catchsim_catch_wait_min, self.config.catchsim_catch_wait_max)
```

Comment:
Same here this simulates waiting for the app to play an animation not waiting for the user to throw the ball.